Changed text-align property in MongoDB Blog

Ramesh623 opened the pull request.

Ramesh623 committed:
Update index.html

Changed text-align property in MongoDB Blog

pankajan05 commented:
You can use this kind of HTML CSS card styles:
Reference link: https://www.enhanzer.com/

![image](https://user-images.githubusercontent.com/37388848/178206555-90818812-ce0b-4a3c-98b1-4d223e530309.png)

![image](https://user-images.githubusercontent.com/37388848/178206888-3e2e08bb-af3a-43b9-8b55-461368f1c206.png)

thenukagunawardana committed:
Updated Logo

Updated the Elzian Agro logo , updated the hover color of the navbar and changed the color of the navbar for easier logo visibility.

thenukagunawardana committed:
Merge branch 'Dev' of https://github.com/Elzian-Agro/Dev-Blog-Site into Dev

thenukagunawardana committed:
Updated MERN Stack blog image

Ramesh623 committed:
Updated MongoDB Blog

Added few css for blog

Ramesh623 committed:
Merge branch 'Dev' of https://github.com/Elzian-Agro/Dev-Blog-Site into Dev

Ramesh623 commented:
Added HTML CSS card styles for MongoDB blog

Ramesh623 committed:
Update styles.css

Ramesh623 committed:
Update styles.css

thenukagunawardana committed:
Updated Contact Us button color

Updated Color from CSS and added content to Contact Us header

SeminiDanoja committed:
Added a blog to portfolio section

SeminiDanoja committed:
Merge branch 'Dev' of https://github.com/Elzian-Agro/Dev-Blog-Site into Dev

Ramesh623 committed:
Added bootstrap grid for MongoDB blog

Ramesh623 committed:
Merge branch 'Dev' of https://github.com/Elzian-Agro/Dev-Blog-Site into Dev

thenukagunawardana committed:
Updated Elzian Agro Logo

Added padding to the logo and centered it a bit

SeminiDanoja committed:
Added content to UI/UX blog in technology section

SeminiDanoja committed:
Merge branch 'Dev' of https://github.com/Elzian-Agro/Dev-Blog-Site into Dev

SeminiDanoja committed:
Update the UI?UX blog

Himaya485 committed:
Added my profile

SeminiDanoja committed:
Added a style for card in technology section

thenukagunawardana committed:
Added Small changes to MERN Blog

thenukagunawardana committed:
Merge branch 'Dev' of https://github.com/Elzian-Agro/Dev-Blog-Site into Dev

thenukagunawardana committed:
Updated Padding on the Elzian Agro Logo

pankajan05 reviewed (approved):
Checked
